hbase/hbase-hbase/src/test/java/cz/o2/proxima/storage/hbase/HBaseLogObservableTest.java

```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright 2017 O2 Czech Republic, a.s.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cz.o2.proxima.storage.hbase;
+
+import com.typesafe.config.ConfigFactory;
+import cz.o2.proxima.repository.AttributeDescriptor;
+import cz.o2.proxima.repository.EntityDescriptor;
+import cz.o2.proxima.repository.Repository;
+import cz.o2.proxima.storage.Partition;
+import cz.o2.proxima.storage.StreamElement;
+import cz.o2.proxima.storage.batch.BatchLogObserver;
+import static cz.o2.proxima.storage.hbase.TestUtil.b;
+import cz.seznam.euphoria.shaded.guava.com.google.common.collect.Lists;
+import java.io.IOException;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+import org.apache.hadoop.hbase.HBaseTestingUtility;
+import org.apache.hadoop.hbase.MiniHBaseCluster;
+import org.apache.hadoop.hbase.TableName;
+import org.apache.hadoop.hbase.client.Connection;
+import org.apache.hadoop.hbase.client.ConnectionFactory;
+import org.apache.hadoop.hbase.client.Table;
+import static org.junit.Assert.*;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Test suite for {@link HBaseLogObservable}.
+ */
+public class HBaseLogObservableTest {
+
+  final Repository repo = Repository.Builder.ofTest(ConfigFactory.load()).build();
+  final EntityDescriptor entity = repo.findEntity("test").get();
+  final AttributeDescriptor<?> attr = entity.findAttribute("dummy").get();
+  final AttributeDescriptor<?> wildcard = entity.findAttribute("wildcard.*").get();
+  final TableName tableName = TableName.valueOf("test");
+
+  HBaseTestingUtility util;
```

Comment:
you should give more attention to property (and method) visibility... all of the followings should be private